tests/gtest/avifgridapitest.cc

```diff
@@ -76,6 +76,11 @@ avifResult EncodeDecodeGrid(const std::vector<std::vector<Cell>>& cell_rows,
   testutil::AvifImagePtr grid = testutil::CreateImage(
       static_cast<int>(image->width), static_cast<int>(image->height),
       /*depth=*/8, yuv_format, AVIF_PLANES_ALL);
+  const int num_rows = (int)cell_rows.size();
+  const int num_cols = (int)cell_rows[0].size();
+  AVIF_CHECKRES(
+      testutil::MergeGrid(num_cols, num_rows, cell_images, grid.get()));
+
```

Comment:
Rows below should be removed now that there is `testutil::MergeGrid()`

Note: `testutil::MergeGrid()` could be made in another PR to reduce the size of this one

Reply:
Indeed I forgot to remove them, thanks for spotting it.
I agree it could be split... but I'm afraid of spending a while fighting with git and merges again... So if it's not a strong request I'd rather leave it as is.